components/Article/BlockQuote/index.module.scss

```diff
@@ -0,0 +1,18 @@
+.blockQuote {
```

Comment:
Do you still need the class, or is the specificity enough if you just style the plain element?

Reply:
I suppose as a CSS module just the element name should be enough as the styles are scoped, but might be worth a test.

Reply:
I believe CSS modules don't allow top level pure elements